toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -81,6 +103,8 @@ do
     chroot "$chroot_builder_folder" rpm --import "$gpg_key"
 done
 
+record_timestamp "finish importing GPG keys"
```

Comment:
Could we record the beginning of each step as well? This would also prevent us from mixing in the time spent on other steps, which we didn't record. In this case we're combining overwriting the RPM database (not measured) with importing the GPG keys.